File: site/comment.php
 <?php
$servername = "localhost";
$username = "root";
$password = "";
$dname="site";
if(isset($_GET['aid']))
{$conn=new mysqli($servername,$username,$password,$dname);
	session_start();

$admin=0;
if(isset($_SESSION['user']))
{$sql="SELECT * FROM user WHERE USERNAME='".$_SESSION['user']."'";
// Check connection
echo $sql;
//0-name 1-username 2-password 3-mail
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
     $result = $conn->query($sql);
    if($result)
	{$row=mysqli_fetch_assoc($result);
     $admin=$row['ADMIN'];
	 echo $admin."admin";
	}} 
 
echo $_GET['aid'];
$sql="SELECT * FROM comment WHERE AID=".$_GET['aid']." ORDER BY DATE DESC ";
echo $sql;
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}
     $result = $conn->query($sql);
    if(!$result)
		echo "No comments";
    else
	{echo $result->num_rows."comments<br>";
    echo "<div class=\"container-fluid\">";
	 while($row = mysqli_fetch_assoc($result))
	{ echo "<div class=\"row row_com\">";
      echo "<div class=\"col-2 \">";
      echo "<p class=\"user_com\">";
      echo $row["UID"]."<br>".$row['DATE'],"</p></div>";
      echo "<div class=\"col-4\">";
      echo "<p class=\"txt_com\">";
      echo $row['TEXT']."<br>";
      echo "<span class=\"point\"><i class=\"fa fa-thumbs-o-up\">up</i></span><span class=\"point\"> <i class=\"fa fa-thumbs-down\">down</i></span>";
      if (isset($_SESSION['user'])&&($_SESSION['user']==$row['UID']||$admin==1))
	  {echo "<span class=\"point\"><i class=\"fa fa-times\" align=\"right\" onClick=\"remcom('".$row['CID']."','".$row['AID']."')\">remove</i></span>";}
      echo "</p></div>";
      echo "</div>";
	   echo "</div>";}
	 echo "</div>";
}
 $result->close();
}
?> 

File: site/scriptaux/comment.py
#!/Python27/python
import cgi, cgitb
import re,os
import MySQLdb

print "Content-type:text/html\r\n\r\n"
#print "<link rel=\"stylesheet\" href=\"bootstrap.css\">"
#print "<script src=\"bootstrap.js\"></script>"
print "<body>"

form = cgi.FieldStorage() 
aid =str(form.getvalue('aid'))
user =str(form.getvalue('user'))
print aid
db = MySQLdb.connect("localhost","root","","site")
cursor = db.cursor()
sql = "SELECT * FROM comment WHERE AID="+aid+" ORDER BY DATE DESC "
#print sql
try:
   # Execute the SQL command
   cursor.execute(sql)
   # Fetch all the rows in a list of lists.
   results = cursor.fetchall()
   #print results
   print "<div class=\"container-fluid\">"
   print user,"<br>"
   for row in results:
      #print row
      print "<div class=\"row row_com\">"
      print "<div class=\"col-2 \">"
      print "<p class=\"user_com\">"
      print row[1],"<br>",row[4],"</p></div>"
      print "<div class=\"col-4\">"
      print "<p class=\"txt_com\">"
      print row[3],"<br>"
      print "<i class=\"fa fa-thumbs-o-up\">up</i> <i class=\"fa fa-thumbs-down\">down</i>"
      print "<i class=\"fa fa-times align=\"right\"\">remove</i>" if user==row[1] and user!="anonymous" else ""
      print "</p></div>"
      print "</div>"
   print "</div>" 
except:
   print "No DataBase"

def getname(uid):
   sql1 = "SELECT * FROM user WHERE aid=\"+aid\""
   print sql1
   try:
      cursor.execute(sql)
      results = cursor.fetchall()
      return results[0][1] 
   except:
      print "No DataBase"
print "</body>"
print "</html>"

